activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/test/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableEntityDeletedHandlerTest.java

```diff
@@ -57,8 +64,10 @@ public void handleRemoveVariableFromProcessAnDeleteIt() {
         variableEntity.setName("var");
         processInstanceEntity.getVariables().add(variableEntity);
 
-        given(entityManager.find(ProcessInstanceEntity.class,
-                                "procInstId")).willReturn(processInstanceEntity);
+        when(entityManager.createEntityGraph(ProcessInstanceEntity.class)).thenReturn(mock(EntityGraph.class));
+        given(entityManager.find(eq(ProcessInstanceEntity.class),
+                                 eq("procInstId"),
+                                 any(Map.class))).willReturn(processInstanceEntity);
```

Comment:
This will be simplified if we use non static methods in `EntityManagerFinder`:
```
given(entityManagerFinder.findProcessInstanceWithVariables(entitiyManager, "procInstId")).willReturn(processInstanceEntity);
```

Reply:
Fixed